indexGrid.html

```diff
@@ -0,0 +1,100 @@
+      <h1 class='navbar-title'>Developer Tools</p>
+    </nav>
+  </header>
+
+  <div class='container'>
+    <h1 class='title'>Front-End Tools</p>
+    <div class='grid-container'>
+
+      <div class='item1'>
+        <div class='cont-logo'><img class="logo" src='assets\react_icon.svg'/></div>
+        <h2 class='name'>ReactJS</h2>
+        <span class='date'>Release: 2013</span>
+        <p class="info">React is a declarative, component-based JavaScript library for building user interfaces.</p>
+      </div>
+
+        <div class='item2'>
+        <div class='cont-logo'><img class="logo" src='assets\angular_icon.svg'/></div>
+        <h2 class='name'>AngularJS</h2>
+        <span class='date'>Release: 2010</span>
+        <p class='info'>AngularJS (commonly refered to as AngularJS 1.X)is a JavaScript framework for building SPAs with an MVC architecture</p>
+      </div>
+
+      <div class='item2'/>
+        <div class='cont-logo'><img class="logo" src='assets\vue_icon.svg'/></div>
+        <h2 class='name'>VueJS</h2>
+        <span class='date'>Release: 2014</span>
+        <p class='info'>VueJS (pronunced/vju:/, like view) is an open-source progressive JavaScript framework for builden user interfaces</p>
+      </div>
+
+      <div class='item2'/>
+        <div class='cont-logo'><img class="logo" src='assets\elm_icon.svg'/></div>
+        <h2 class='name'>Elm</h2>
+        <span class='date'>Release: 2016</span>
+        <p class='info'>Elm is a domain-specific functional programming language for declaratively creating web browser-based graphical user interfaces</p>
+      </div>
+
+      <div class='item2'/>
+        <div class='cont-logo'><img class="logo" src='assets\polymer_icon.svg'/></div>
+        <h2 class='name'>Polymer</h2>
+        <span class='date'>Release: 2015</span>
+        <p class='info'>Polymer is an open-source JavaScript library for builden web applications using Web Components</p>
+      </div>
+
+    </div>
+  </div>
+
+  <div class='container'>
+    <p class='title'>Back-end tools</p>
+    <div class='grid-container'>
+
+      <div class='item1'>
+        <div class='cont-logo'><img class="logo" src='assets\rails_icon.png'/></div>
+        <h2 class='name'>Ruby on Rails</h2>
+        <span class='date'>Release: 2005</span>
+        <p class='info'>Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+      </div>
+
+      <div class='item2'>
+        <div class='cont-logo'><img class="logo" src='assets\express_icon.svg'/></div>
+        <h2 class='name'>NodeJS</h2>
+        <span class='date'>Release: 2009</span>
+        <p class='info'>Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Uses an event-driven, non-blocking I/O model that makes it lightweight and efficient.</p>
+      </div>
+
+      <div class='item2'>
```

Comment:
There's no need to change to `item-1` and `item-2`, you can just use the pseudo-class `:first-child` to change the technologies that are needed and the placement can be done by the css-grid template alone :smile: